content/webapp/pages/stories.js

```diff
@@ -24,10 +23,28 @@ import SpacingComponent from '@weco/common/views/components/SpacingComponent/Spa
 import Space from '@weco/common/views/components/styled/Space';
 import { staticBooks } from '../content/static-books';
 import { FeaturedCardArticle } from '@weco/common/views/components/FeaturedCard/FeaturedCard';
+import styled from 'styled-components';
+import { prismicPageIds } from '@weco/common/services/prismic/hardcoded-id';
+import FeaturedTextTitle from '@weco/common/views/components/FeaturedText/FeaturedText';
+import { defaultSerializer } from '@weco/common/services/prismic/html-serializers';
+import {
+  getPage,
+  getPageFeaturedText,
+} from '@weco/common/services/prismic/pages';
+import { type FeaturedText } from '@weco/common/model/text';
+const Header = styled.h1.attrs(props => ({
```

Comment:
Is there a way we can de-dupe this (also same/similar in `PageHeader`) especially since it introduces custom font sizes?

Reply:
We can move this out in a style components layout, which i thought the same. What are your thoughts on this ? Alternatively creating a new class name with the custom font sizing but i know what we want to go with more with styled components

Reply:
we can move this to common/styled/ sectionHeader? what you guys think and then refer this to here ?

Reply:
ive moved this into here common/styled/